packages/cubejs-schema-compiler/scaffolding/ScaffoldingTemplate.js

```diff
@@ -75,7 +75,7 @@ class ScaffoldingTemplate {
   }
 
   memberName(member) {
-    return inflection.camelize(member.title.replace(/\s+/g, '_').toLowerCase(), true);
+    return inflection.camelize(member.title.replace(/\s|\.+/g, '_').toLowerCase(), true);
```

Comment:
I believe it should be `/\s+|\./g`. In this multiple space rule kept intact.

Reply:
ya right . Thanks